src/test/java/uk/nhs/hee/tis/trainee/forms/service/FormRPartAServiceImplTest.java

```diff
@@ -242,4 +314,13 @@ void shouldGetDraftFormRPartAById() {
     assertThat("Unexpected forename.", dto.getForename(), is(DEFAULT_FORENAME));
     assertThat("Unexpected surname.", dto.getSurname(), is(DEFAULT_SURNAME));
   }
+
+  private static FormRPartA copyForm(FormRPartA input) {
+    FormRPartA output = new FormRPartA();
+    output.setId(input.getId());
+    output.setTraineeTisId(input.getTraineeTisId());
+    output.setForename(input.getForename());
+    output.setSurname(input.getSurname());
+    return output;
+  }
```

Comment:
If I have to find something, you could reduce the size of the class be replacing this with a call to `org.springframework.beans.BeanUtils.copyProperties(input, new FormRPartA())`.

Reply:
Nice, didn't know of that util. I'll make the changes.